docs/.storybook/configs/contexts.js

```diff
@@ -1,8 +1,14 @@
 import intlContext from './intl-context';
 import themeContext from './theme-context';
 
+// We only want to show the theme switcher in local environment
+// or in preview deployments
+const shouldIncludeThemeContext = () =>
+  process.env.NODE_ENV !== 'production' ||
+  window.location.hostname.includes('commercetools.vercel.app');
```

Comment:
I tried using the `VERCEL_ENV` env variable but it didn't work so I used this alternative.